hyperactor/Cargo.toml

```diff
@@ -54,7 +54,7 @@ serde_yaml = "0.9.25"
 signal-hook-tokio = { version = "0.3", features = ["futures-v0_3"] }
 thiserror = "2.0.12"
 tokio = { version = "1.46.1", features = ["full", "test-util", "tracing"] }
-tokio-rustls = { version = "0.24.1", features = ["dangerous_configuration"] }
+tokio-rustls = { git = "https://github.com/shayne-fletcher/tokio-rustls", rev = "62b6a48e4c14a05c193508b9d98a0be6b0cb4baa", features = ["dangerous_configuration"] }
```

Comment:
I don't think we should use personal repos for this. We can create a clone either in `pytorch-labs` or in `facebook/experimental`